server/src/locale_en_US.json

```diff
@@ -1,21 +1,23 @@
 {
-	"srr_group-assets" : "Assets management", 
-	"srr_security-wallet" : "Security Wallet",
-	"srr_asset-agent" : "Physical assets",
-	"srr_virtual-assets" : "Virtual assets",
-	"srr_alert-agent" : "Alert settings",
-	"srr_automation-settings" : "Automation settings",
-	"srr_automations" : "Automations",
-	"srr_group-discovery" : "Auto discovery settings",
-	"srr_discovery" : "Auto discovery settings",
-	"srr_group-mass-management" : "Mass management settings",
-	"srr_etn-mass-management" : "Mass management settings",
-	"srr_group-monitoring-feature-name" : "SNMP settings",
-	"srr_monitoring" : "SNMP settings",
-	"srr_group-network" : "Network settings",
-	"srr_network" : "Network settings",
-	"srr_group-notification-feature-name" : "Notification settings",
-	"srr_notification" : "Email settings",
-	"srr_group-user-session-feature-name" : "User session settings",
-	"srr_user-session" : "User session settings"
+  "srr_group-assets" : "Assets management",
+  "srr_security-wallet" : "Security Wallet",
+  "srr_asset-agent" : "Physical assets",
+  "srr_virtual-assets" : "Virtual assets",
+  "srr_alert-agent" : "Alert settings",
+  "srr_automation-settings" : "Automation settings",
+  "srr_automations" : "Automations",
+  "srr_group-discovery" : "Auto discovery settings",
+  "srr_discovery" : "Auto discovery settings",
+  "srr_group-mass-management" : "Mass management settings",
+  "srr_etn-mass-management" : "Mass management settings",
+  "srr_group-monitoring-feature-name" : "SNMP settings",
+  "srr_monitoring" : "SNMP settings",
+  "srr_group-network" : "Network settings",
+  "srr_network" : "Network settings",
+  "srr_group-notification-feature-name" : "Notification settings",
+  "srr_notification" : "Email settings",
+  "srr_group-user-session-feature-name" : "User session settings",
+  "srr_user-session" : "User session settings",
+  "srr_group-user-session-management" : "User session management",
```

Comment:
i don't understand the naming rules here, (between group or single item, with srr_etn prefix or not)...
Anyway, is it normal to have two members *user-session* with the same "User session settings" value here?